convert2rhel/utils.py

```diff
@@ -32,6 +32,39 @@
 from six import moves
 
 
+try:
+    from functools import cmp_to_key
+except ImportError:
+    # Taken from https://github.com/python/cpython/blob/3.10/Lib/functools.py#L206
+    def cmp_to_key(mycmp):
+        """Convert a cmp= function into a key= function"""
+
+        class K(object):
+            __slots__ = ["obj"]
+
+            def __init__(self, obj):
+                self.obj = obj
+
+            def __lt__(self, other):
+                return mycmp(self.obj, other.obj) < 0
+
+            def __gt__(self, other):
+                return mycmp(self.obj, other.obj) > 0
+
+            def __eq__(self, other):
+                return mycmp(self.obj, other.obj) == 0
+
+            def __le__(self, other):
+                return mycmp(self.obj, other.obj) <= 0
+
+            def __ge__(self, other):
+                return mycmp(self.obj, other.obj) >= 0
+
+            __hash__ = None
+
+        return K
```

Comment:
@abadger, I took this code from python 3.10 functools module, should I do the same as you did here? https://github.com/oamg/convert2rhel/pull/540/files#diff-f82efabf7bb126100afd0472e3d8a5902da6576c6ddf75e3184cce9eb647c52aR58-R59

Reply:
@r0x0d Yes, that would be good. Keeping track of licenses and copyright when copying code is important.